src/lib/coda_lib/coda_lib.ml

```diff
@@ -15,7 +15,18 @@ module Ledger_transfer = Ledger_transfer.Make (Ledger) (Ledger.Db)
 module Config = Config
 module Subscriptions = Coda_subscriptions
 
-type processes = {prover: Prover.t; verifier: Verifier.t}
+exception Snark_worker_error of int
```

Comment:
Needs to get added to the `mli` too or else this exception can't be handled outside this module, right?